### PR TITLE
Add more monitoring metrics

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add more monitoring metrics: rewards, numApps, and peer reorg traffic metrics
- Add grafana panels for the new metrics

Related issues:

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM!

File: p2p/index.js [hunk 1]
```diff
@@ -310,17 +310,25 @@ class P2pClient {
...
+          TrafficEventTypes.PEER_REORG_CANDIDATES_BIDIRECTED, bidirectedConnections.length);
       this.pickRandomPeerAndDisconnect(bidirectedConnections);
     }
   }
 
   async tryReorgPeerConnections() {
-    if (Object.keys(this.outbound).length < NodeConfigs.PEER_REORG_MIN_OUTBOUND) {
+    const numOutbound = Object.keys(this.outbound).length;
+    if (numOutbound < NodeConfigs.PEER_REORG_MIN_OUTBOUND) {
+      trafficStatsManager.addEvent(TrafficEventTypes.PEER_REORG_BELOW_MIN_OUTBOUND, numOutbound);
```
Comment (liayoo) [inline p2p/index.js:331]: 👍👍

File: consensus/index.js [hunk 2]
```diff
@@ -1374,6 +1366,12 @@ class Consensus {
       validators,
       globalTimeSyncStatus,
     };
+    const rewards = this.node.getRewards();
```
Comment (liayoo) [inline consensus/index.js:1369]: thanks for adding this!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the fast review. Hope to see the metrics in the next rounds of sandbox experiments! @minsulee2 @liayoo

File: consensus/index.js (hunk 2 above)
Comment (platfowner) [reply consensus/index.js:1369]: thanks!

**Pull request merged**